### DISCUSSION TITLE
How to define different spec.remoteWrite configurations for each Prometheus replica

### DISCUSSION THREAD

**Discussion by anoop2503** [prometheus-operator/prometheus-operator#5734] [Q&A] [answered]
Hi,

I am running two Prometheus replicas and I want to enable remotewrite configurations with different endpoint for each Prometheus replica. Is there anyway to achieve this?

**Prometheus-0:**
```
remote_write:
    - url: http://remote_1_endpoin/api/v1/samples
```

**Prometheus-1:**
```
remote_write:
    - url: http://remote_2_endpoin/api/v1/samples
```

**Comment by ArthurSens** [comment, marked as answer]
By definition of replicas, they are identical, so no 😅 

You could create 2 Prometheus resources with different remote-write configurations though

**Reply by anoop2503** [reply]
Got it. Thanks @ArthurSens.
